src/java.base/share/classes/sun/security/ssl/StatusResponseManager.java

```diff
@@ -687,41 +579,9 @@ private void addToCache(CertId certId, ResponseCacheEntry entry) {
             }
         }
 
-        /**
-         * Determine the delay to use when scheduling the task that will
-         * update the OCSP response.  This is the shorter time between the
-         * cache lifetime and the nextUpdate.  If no nextUpdate is present
-         * in the response, then only the cache lifetime is used.
-         * If cache timeouts are disabled (a zero value) and there's no
-         * nextUpdate, then the entry is not cached and no rescheduling
-         * will take place.
-         *
-         * @param nextUpdate a {@code Date} object corresponding to the
-         *      next update time from a SingleResponse.
-         *
-         * @return the number of seconds of delay before the next fetch
-         *      should be executed.  A zero value means that the fetch
-         *      should happen immediately, while a value less than zero
-         *      indicates no rescheduling should be done.
-         */
-        private long getNextTaskDelay(Date nextUpdate) {
-            long delaySec;
-            int lifetime = getCacheLifetime();
-
-            if (nextUpdate != null) {
-                long nuDiffSec = (nextUpdate.getTime() -
-                        System.currentTimeMillis()) / 1000;
-                delaySec = lifetime > 0 ? Long.min(nuDiffSec, lifetime) :
-                        nuDiffSec;
-            } else {
-                delaySec = lifetime > 0 ? lifetime : -1;
-            }
-
-            return delaySec;
-        }
     }
 
-    static final StaplingParameters processStapling(
+    static StaplingParameters processStapling(
```

Comment:
Even though a static method is implicitly final, the final keyword does prevent subclasses from inadvertently using the same method signature.

Reply:
Did you mean a subclass can declare a static method with the same signature?  I tried an example and the compiling failed.  Did I miss something?

```
class Base {	
	public static void display() {
		System.out.println();
	}
	
	public void print() {
		System.out.println();
	}
}

class Derived extends Base {
//	public void display() {
         public static void display() {
		System.out.println();
	}
	
	public void print() {
		System.out.println();
        }
}
```


```
XXX: error: display() in Derived cannot override display() in Base
	public void display() {
	            ^
  overridden method is static
```